src/main/resources/db/migration/v0.4.0.2__agr_curation_api.sql

```diff
@@ -0,0 +1,230 @@
+CREATE TABLE IF NOT EXISTS loggedinperson (
+	id bigint PRIMARY KEY,
+	oktaemail varchar(255),
+	oktaid varchar(255),
+	apitoken varchar(255)
+	);
+	
+INSERT INTO loggedinperson (id)
+	SELECT id FROM person WHERE id IS NOT NULL;
+	
+UPDATE loggedinperson
+	SET oktaemail = person.email
+	FROM person
+	WHERE loggedinperson.id = person.id;
+	
+UPDATE loggedinperson
+	SET apitoken = person.apitoken
+	FROM person
+	WHERE loggedinperson.id = person.id;
+	
+UPDATE person 
+	SET uniqueid = CONCAT(person.firstname, '|', person.lastname, '|', oktaemail)
+	FROM loggedinperson
+	WHERE loggedinperson.id = person.id;
+	
+ALTER TABLE person
+	ADD COLUMN IF NOT EXISTS modentityid varchar(255);
+	
+UPDATE person
+	SET modentityid = modid;
+	
+ALTER TABLE person
+	DROP COLUMN IF EXISTS email,
+	DROP COLUMN IF EXISTS apitoken,
+	DROP COLUMN IF EXISTS modid;
+	
+ALTER TABLE association
+	DROP COLUMN IF EXISTS created,
+	DROP COLUMN IF EXISTS lastupdated;
+
+ALTER TABLE biologicalentity
+	DROP COLUMN IF EXISTS created,
+	DROP COLUMN IF EXISTS lastupdated;
+	
+ALTER TABLE bulkload
+	ADD COLUMN IF NOT EXISTS created timestamp without time zone,
+	ADD COLUMN IF NOT EXISTS lastupdated timestamp without time zone,
```

Comment:
I don't see these columns in any of the AUD tables.  As for other tables, I don't think it makes sense to copy over the timestamps for all of them - the @UpdateTimestamp and @CreationTimestamp appear to overwrite non-null values so you end up with the time they were loaded into the database rather than any submitted values.